docs/VictoriaLogs/vmalert.md

```diff
@@ -0,0 +1,174 @@
+---
+weight: 10
+title: vmalert
+menu:
+  docs:
+    parent: "victorialogs"
+    weight: 10
+aliases:
+- /VictoriaLogs/vmalert.html
+---
+VictoriaLogs provides log stats APIs [`/select/logsql/stats_query`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-stats)  and [`/select/logsql/stats_query_range`](https://docs.victoriametrics.com/victorialogs/querying/#querying-log-range-stats), which return the log stats in the format compatible with [Prometheus querying API](https://prometheus.io/docs/prometheus/latest/querying/api/#instant-queries). This allows users using VictoriaLogs as the datasource of [vmalert](https://docs.victoriametrics.com/vmalert/) and creating alerting and recording rules using [LogsQL](https://docs.victoriametrics.com/victorialogs/logsql/).
+
+Note: This page only provides integration instructions for vmalert with VictoriaLogs, see full textbook of vmalert [here](https://docs.victoriametrics.com/vmalert).
+
+## Quick Start
+
+Run vmalert with the `-rule.defaultRuleType=vlogs` flag.
+```
+./bin/vmalert -rule=alert.rules \            # Path to the file with rules configuration.
+    -datasource.url=http://localhost:9428 \  # VictoriaLogs address.
+    -rule.defaultRuleType=vlogs \            # Change default rule type to VictoriaLogs.
+    -notifier.url=http://localhost:9093 \    # AlertManager URL (required if alerting rules are used)
+    -remoteWrite.url=http://localhost:8428 \ # Remote write compatible storage to persist rules and alerts state info (required if recording rules are used)
+    -remoteRead.url=http://localhost:8428 \  # Prometheus HTTP API compatible datasource to restore alerts state from
+```
+
+See the full list of configuration flags in [configuration](#configuration) section.
+
+Each rule file may contain arbitrary number of [groups](https://docs.victoriametrics.com/vmalert/#groups), examples can be found in [Groups](#groups) section.
+
+```yaml
+groups:
+  [ - <rule_group> ]
+```
+> _To validate the syntax of configured rules simply run vmalert with `-rule` and `-dryRun` cmd-line flags._
```

Comment:
I'd skip these, as it doesn't provide much useful info. It is better to mention full group config link in examples below.

Reply:
Removed.